xcape/CAPE_CODE_pressure_lev.f90

```diff
@@ -82,7 +82,91 @@ subroutine loopcape_pl(p3d_in , t3d_in , td3d_in, ps1d_in , ts1d_in , tds1d_in,
     enddo
     return
     end subroutine loopcape_pl
+!-----------------------------------------------------------------------
+!ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+!-----------------------------------------------------------------------
+    subroutine loopcape_pl1d(t3d_in , td3d_in, p3d_in , ps1d_in , ts1d_in , tds1d_in, &
+                        &pinc, source, ml_depth, adiabat, start_3d, &
```

Comment:
note how the order of the variables going in is different:
from p,t,td to t,td,p
this is necessary to have a pyf file that was correctly generated automatically in the optional part as well.